### PR TITLE
Pixel: update layout settings

### LINKED ISSUE #6458: Pixel: Styles: Layout
Update spacing and layout styles.

Designs: Cm51iCQSPu8Ar1qufDTeOQ-fi

### PR DESCRIPTION
**Pull request by madhusudhand**
#### Changes proposed in this Pull Request:

Add fluid outer spacing.
Move shadow to custom variable.

Before:

| Before | After |
| --- | --- |
| image | image |
| normal, wide, full width |  |
| image | image |

#### Related issue(s):

Fixes: #6458

### CODE REVIEW

File: pixel/style.css [hunk 1]
```diff
@@ -20,5 +20,16 @@ Tags: one-column, custom-colors, custom-menu, custom-logo, editor-style, feature
  * Need this until box shadow support added to Gutenberg here: https://github.com/WordPress/gutenberg/pull/43184 
  */
 .pixel-shadow {
-    box-shadow: 5px 5px 0px -2px var(--wp--preset--color--background), 5px 5px var(--wp--preset--color--foreground);
+    box-shadow: var(--wp--custom--shadow)
```
Comment (jffng) [inline pixel/style.css]:
```suggestion
    box-shadow: var(--wp--custom--shadow);
```

File: pixel/style.css [hunk 2]
```diff
@@ -20,5 +20,16 @@ Tags: one-column, custom-colors, custom-menu, custom-logo, editor-style, feature
...
+}
+
+
+/*
+ * Margin resets for full-width blocks
+ */ 
+.wp-site-blocks .alignfull {
```
Comment (jffng) [inline pixel/style.css]: Should we try to use the `useRootPaddingAwareAlignments` setting? In theory with that setting enabled, we shouldn't need this CSS.

Comment (madhusudhand) [reply pixel/style.css]: Thanks for the suggestion. enabled the above setting that eliminates the custom css. 👍

**Comment by madhusudhand** [comment]
Merging this to avoid dependencies with other items.

**Pull request merged**